.github/SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security policy
+
+## Supported Versions
+
+Only versions of the providers that target versions of ASP.NET Core that are in support by Microsoft are supported by the maintainers.
+
+The .NET Core and ASP.NET Core support policy, including currently supported versions, can be found at the [.NET and .NET Core Support Policy page](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
+
+## Reporting a Vulnerability
+
+Security issues and bugs should be reported privately by emailing CHANGE_ME.
```

Comment:
If you don’t mind adding my personal address, we can use contact@kevinchalet.com.

Reply:
That's fine with me - I just didn't want it to be my own personal email 😄